libr/core/cmd_print.c

```diff
@@ -1701,11 +1713,11 @@ static void annotated_hexdump(RCore *core, const char *str, int len) {
 	rows = len / nb_cols;
 
 	chars = calloc (nb_cols * 40, sizeof (char));
-	if (!chars) goto err_chars;
+	if (!chars) { goto err_chars; }
 	note = calloc (nb_cols, sizeof (char *));
-	if (!note) goto err_note;
+	if (!note) { goto err_note; }
 	bytes = calloc (nb_cons_cols * 40, sizeof (char));
-	if (!bytes) goto err_bytes;
+	if (!bytes) { goto err_bytes; }
```

Comment:
there are no oneliner ifs in the CODINGSTYLE of r2. please add newlines after { and ;